libraries/Filter/NotchFilter.cpp

```diff
@@ -72,30 +90,30 @@ const AP_Param::GroupInfo NotchFilterParams::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO_FLAGS("ENABLE", 1, NotchFilterParams, _enable, 0, AP_PARAM_FLAG_ENABLE),
 
+    // @Param: ATT
+    // @DisplayName: Attenuation
+    // @Description: Notch attenuation in dB
+    // @Range: 5 30
+    // @Units: dB
+    // @User: Advanced
+    AP_GROUPINFO("ATT", 4, NotchFilterParams, _attenuation_dB, 15),
+
     // @Param: FREQ
     // @DisplayName: Frequency
     // @Description: Notch center frequency in Hz
     // @Range: 10 400
     // @Units: Hz
     // @User: Advanced
-    AP_GROUPINFO("FREQ", 2, NotchFilterParams, _center_freq_hz, 80),
```

Comment:
a comment on why 2 and 3 are unused (and reserved) would be good

Reply:
Done